data_science_python.ipynb

```diff
@@ -762,15 +765,52 @@
      "cell_type": "markdown",
      "metadata": {},
      "source": [
+      "We will use english stop words in our count vectorizer. For that we need to download the stopwords corpora."
```

Comment:
Why not use `gensim.parsing.preprocessing.STOPWORDS` instead? No need to introduce a beast like NLTK.

But the downloaded stopwords are not used at all, so I'm not sure why this is even here.